lib/libjpeg/jconfigint.h

```diff
@@ -0,0 +1,31 @@
+/* libjpeg-turbo build number */
+#define BUILD  "20191111"
+
+/* Compiler's inline keyword */
+#undef inline
```

Comment:
What is this for? This looks.. weird?
It implies that someone previously defined `inline` (which should never happen).

Reply:
No idea. It's part of the template from which `jconfigint.h` gets auto generated (same above).

Reply:
Fair enough